isrcsubmit.py

```diff
@@ -43,6 +43,8 @@
 from optparse import OptionParser
 from subprocess import Popen, PIPE, call
 
+os.add_dll_directory(os.getcwd())
```

Comment:
I am in Python 64-bit v3.7.4 in Windows 10, this row throws following error and prevents isrcsubmit from running:

    AttributeError: module 'os' has no attribute 'add_dll_directory'

What could we do to fix script for version 3.8 without breaking older versions?

Reply:
Same with Python 32-bit v3.4.3 in Windows XP.

(both setups are running fine without this line)